package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appium-adb",
-  "version": "6.25.0",
+  "version": "6.25.1",
```

Comment:
the module version should not be set manually. It is bumped as a part of publishing process

Reply:
Wops! Fixed